configs/razanne.toml

```diff
@@ -1,8 +1,8 @@
 [atlas]
-    numRobots                = 20
+    numRobots                = 15
     floorplan                = "scattered_obstacles"           # filename in maps
     navigationAlgorithm      = "Atlas"                         # options: Atlas, Ballistic
-    relayAlgorithm           = ["Recovery", "SelfHealing"]     # options: Recovery, Naive, SelfHealing
+    relayAlgorithm           = "Recovery"                      # options: Recovery, SelfHealing, None
```

Comment:
what does "None" do? No relays? If that is the case, rename to `NoRelays` (`None` is something in Python, it's confusing to use this name)